benchmarks/GraphBLAS-sharp.Benchmarks/BenchmarksEWiseAdd.fs

```diff
@@ -0,0 +1,220 @@
+namespace GraphBLAS.FSharp.Benchmarks
+
+open System.IO
+open GraphBLAS.FSharp
+open GraphBLAS.FSharp.Predefined
+open GraphBLAS.FSharp.MatrixBackend
+open BenchmarkDotNet.Attributes
+open BenchmarkDotNet.Engines
+open BenchmarkDotNet.Configs
+open BenchmarkDotNet.Columns
+open BenchmarkDotNet.Filters
+open BenchmarkDotNet.Jobs
+open BenchmarkDotNet.Order
+open Brahma.FSharp.OpenCL.WorkflowBuilder.Basic
+open Brahma.FSharp.OpenCL.WorkflowBuilder.Evaluation
+open OpenCL.Net
+
+module Float32Monoid =
```

Comment:
Should it be moved to our collection of semirings? Yes, this example is pretty simple, but let it be yet another example. Like trivial Boolean semiring.